ws-test-harness/core/pom.xml

```diff
@@ -17,6 +17,7 @@
         <groupId>org.glassfish.metro</groupId>
         <artifactId>ws-test-harness</artifactId>
         <version>2.4.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
```

Comment:
Do you need this? Isn't that the default?

Reply:
Yes. To reproduce what is going on without this line:

-clone the repo, delete local mvn cache (or use some empty one)
-cd into the project folder and do mvn install)

=> fails because 'org.glassfish.metro:ws-test-harness:2.4.0-SNAPSHOT cannot be found